regolith/schemas.py

```diff
@@ -10,17 +11,20 @@
 
 ACTIVITIES_TYPES = ["teaching", "research"]
 AGENCIES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused", "cancelled"]
```

Comment:
this is not right.  You should be importing and using the already existing PROJECTUM_ACTIVE_STATI and so on.